docker-compose.yml

```diff
@@ -145,10 +145,14 @@ services:
   quoting-service:
```

Comment:
Don't we need two services for quoting? Api service and handler service?

Reply:
@vijayg10 Since the API server, the handlers and the monitoring servers can now be run on a single machine, having a separate service for the handler is purely optional, except, maybe in perf test setup.